packages/react-component-library/src/styled-components/partials/StyledOuterWrapper.tsx

```diff
@@ -42,7 +42,6 @@ export const StyledOuterWrapper = styled.div<StyledOuterWrapperProps>`
       return css`
         ${defaults};
         border: ${BORDER_WIDTH[$size]} solid ${color('action', '500')};
-        box-shadow: 0 0 0 ${BORDER_WIDTH[$size]} ${color('action', '100')};
```

Comment:
Did you mean to remove this? Will need to add it on the individual components then (TextInputE etc.)